### DISCUSSION TITLE
escreva já com \n

### DISCUSSION THREAD

**Discussion by WilCb** [dgadelha/Portugol-Webstudio#249] [Ideias e sugestões]
Boa noite pessoal, gostaria de sugerir um `escreva()` já com `"\n"` para pular linha

Aqui está meu código de exemplo que funciona perfeitamente

```
funcao vazio escreval(cadeia texto) {
		/*

		Escreve igual ao escreva() pórem já pula a linha
		
		*/
		cadeia pulaLinha = texto + "\n"
		escreva(pulaLinha)
	}
```

**Comment by dgadelha** [comment]
O mesmo caso desse outro comentário aqui: https://github.com/dgadelha/Portugol-Webstudio/discussions/167#discussioncomment-8620950

A função `escreval` seria interessante, mas ainda não acho que temos recursos para nos separar do Portugol Studio...

**Comment by matthedumguy** [comment]
Eu fiz algo que ajuda mais ou menos,
  funcao escreva_ext(cadeia txt){
    retorne escreva(txt, "\n") 
  }
  
  mais assim tenho que usar + ao inves de ,

**Comment by mastergig** [comment]
Em C#, o comando é Console.WriteLine, creio que L não faz juz ao significado, escrevalinha é melhor
